composeApp/src/commonMain/kotlin/domain/Event.kt

```diff
@@ -0,0 +1,19 @@
+package domain
+
+data class Event (
```

Comment:
could we format this class, please? `Cmd + Shift  + L` could work

There are some double blank spaces

Reply:
done